config/views.yaml

```diff
@@ -51,10 +51,56 @@ component_readiness:
       pity_factor: 5
       ignore_missing: false
       ignore_disruption: true
+      pass_rate_required_new_tests: 95
```

Comment:
This I'd like to enable and I don't think is too controversial. It's two additional regressions today.

Reply:
This morning there were two, now there are none below 95%, you can set to 99% to see/test the mixed response in advanced options if none are presently showing up.